docs/content/querying/lookups.md

```diff
@@ -28,6 +28,263 @@ and such data belongs in the raw denormalized data for use in Druid.
 Very small lookups (count of keys on the order of a few dozen to a few hundred) can be passed at query time as a "map" 
 lookup as per [dimension specs](../querying/dimensionspecs.html).
 
-For lookups defined cluster-wide rather than embedded in the query, please look at the [namespaced lookup extension](../development/extensions-core/namespaced-lookup.html).
-
 For other additional lookups, please see our [extensions list](../development/extensions.html).
+
+Dynamic configuration (EXPERIMENTAL)
```

Comment:
I can, but don't know the syntax you're looking for, is there another doc that uses that indication?